.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
No actually no actually the one on my pc only have one folder inside photo i.e. `cache` which shouldn't be tracked, the images are fine ?
![screenshot 17](https://user-images.githubusercontent.com/21199234/26935549-e4bff0e2-4c89-11e7-8726-1980f04811a4.png)